argo-cd-apps/base/all-clusters/infra-deployments/tracing-workload-otel-collector/tracing-workload-tracing.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: argoproj.io/v1alpha1
 kind: ApplicationSet
 metadata:
-  name: enable-tekton-tracing
+  name: tracing-workload-tracing
```

Comment:
If you want to not have this application in prod yet, you need to delete it from production overlay as well as internal production
```
diff --git a/argo-cd-apps/overlays/konflux-public-production/delete-applications.yaml b/argo-cd-apps/overlays/konflux-public-production/delete-applications.yaml
index 832e022f..1272f9f7 100644
--- a/argo-cd-apps/overlays/konflux-public-production/delete-applications.yaml
+++ b/argo-cd-apps/overlays/konflux-public-production/delete-applications.yaml
@@ -12,6 +12,13 @@ metadata:
   name: tracing-workload-otel-collector
 $patch: delete
 ---
+# otel-collector is dev only.
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: tracing-workload-tracing
+$patch: delete
+---
 # mintmaker not ready for prod yet
 apiVersion: argoproj.io/v1alpha1
 kind: ApplicationSet
diff --git a/argo-cd-apps/overlays/production-downstream/delete-applications.yaml b/argo-cd-apps/overlays/production-downstream/delete-applications.yaml
index 390826cc..eb350ebd 100644
--- a/argo-cd-apps/overlays/production-downstream/delete-applications.yaml
+++ b/argo-cd-apps/overlays/production-downstream/delete-applications.yaml
@@ -36,3 +36,17 @@ kind: ApplicationSet
 metadata:
   name: mintmaker
 $patch: delete
+---
+# otel-collector is dev only.
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: tracing-workload-otel-collector
+$patch: delete
+---
+# otel-collector is dev only.
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: tracing-workload-tracing
+$patch: delete

```

Reply:
Deleted!